WooCommerce/Classes/ViewRelated/Dashboard/Settings/Domains/DomainSettingsListView.swift

```diff
@@ -0,0 +1,79 @@
+import SwiftUI
+
+/// Shows a list of domains in domain settings with a CTA to add a domain.
+struct DomainSettingsListView: View {
+    let domains: [DomainSettingsViewModel.Domain]
+    let addDomain: () -> Void
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: 0) {
+            Text(Localization.header.uppercased())
+                .padding(Layout.headerPadding)
+                .foregroundColor(Color(.secondaryLabel))
+                .captionStyle()
+
+            ForEach(domains, id: \.name) { domain in
+                HStack {
```

Comment:
❓ Are we planning to add some button on the right side of the item? Asking because I'm not sure why we need a HStack to embed the contents here.

Reply:
Good question, we don't plan to add any button to the trailing side. Originally it was the only straightforward way for me to align the domain rows on the trailing edge, however it doesn't look needed anymore. Removed in https://github.com/woocommerce/woocommerce-ios/pull/8606/commits/5ba1ff52ce9c99ee08c0368e517346267f31972f